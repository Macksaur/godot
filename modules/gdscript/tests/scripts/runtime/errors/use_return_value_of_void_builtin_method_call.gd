func test():
	var value := []
	print(value.reverse())
