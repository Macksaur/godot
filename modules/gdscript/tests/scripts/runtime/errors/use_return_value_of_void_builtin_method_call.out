GDTEST_OK
>> WARNING
>> Line: 3
>> RETURN_VALUE_VOID
>> Cannot get return value of call to "reverse()" because it returns "void".
<null>
